app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+    validates :username, uniqueness: true
+    validates :email, uniqueness: true, presence: true, email: true
+    validates :name, presence: {string: true}, length:{minimum: 5}
```

Comment:
`[OPTIONAL]` 
 - [ ] making the name of minimum 5 characters I think is not a good idea. 
What if John wants to create an account or Joe or Nick :smile:

Reply:
Perfect I changed it for two